qt/scientific_interfaces/ISISReflectometry/GUI/RunsTable/RunsTablePresenter.cpp

```diff
@@ -452,6 +452,7 @@ void RunsTablePresenter::updateGroupName(MantidWidgets::Batch::RowLocation const
     cell.setContentText(oldValue);
     m_view->jobs().setCellAt(itemIndex, column, cell);
   }
+  m_mainPresenter->notifyGroupNameChanged(m_model.mutableReductionJobs().mutableGroups()[0]);
```

Comment:
Should this use `groupIndex` rather than `0`?

Reply:
Yes, it should. I think I picked that up in the next PR but forgot to come back and amend it here 🤦